docs/en/modules/node.md

```diff
@@ -145,6 +145,36 @@ none
 #### Returns
 flash ID (number)
 
+## node.flashindex()
+
+Returns the function reference for a function in the LFS (Lua Flash Store).
+
+#### Syntax
+`node.flashindex()`
+
+#### Parameters
+`modulename`  The name of the module to be loaded.  If this is `nil` or invalid then an info list is returned
```

Comment:
Would it not be more friendly to raise an error and/or return `nil` if you give it an invalid `modulename`? I don't see the info list as being what anyone would really want if they provide a typo'd module name.

Reply:
This follows the same convention as the Lua `pcall()` and in practice simplifies Lua coding.  I don't really see users calling this directly, but rather the `_init` stub in LFS will set up the environment, e.g.
```Lua
do
  local index = node.flashindex 
  local lfs_t = {
    __index = function(_, name)
        local fn_ut, ba, ma, size, modules = index(name)
        if not ba then 
          return fn_ut
        elseif name == '_time' then
          return fn_ut
        elseif name == '_config' then
          local fs_ma, fs_size = file.fscfg()
          return {ba, ma, fs_ma, size, fs_size}
        elseif name == '_list' then
          return modules
        else
          return nil
        end
            end,
    __newindex = function(_, name, value)
        error("LFS is readonly. Invalid write to LFS." .. name, 2)
      end
  }
  rawset(getfenv(),'LFS', setmetatable(lfs_t,lfs_t))

  local function loader_flash(module)
    local fn, ba = index(module)
    return ba and "Module not in LFS" or fn 
  end
  package.loaders[3] = loader_flash

end
```
now `LFS.somemodule(x,y,z)` just works and the LFS properties `_time`, `_config` and `_list` access the other bits that you need.

I'll explain this example in the LFS paper.